2019-03-18 18:08:53,128 : INFO : Machine : Mission Pinball Framework Core Engine v0.50.12
2019-03-18 18:08:53,129 : INFO : Machine : Command line arguments: {'no_load_cache': False, 'create_config_cache': True, 'bcp': True, 'configfile': ['config.yaml'], 'mpfconfigfile': 'c:\\users\\wilder\\appdata\\local\\programs\\python\\python36\\lib\\site-packages\\mpf\\mpfconfig.yaml', 'force_assets_load': False, 'logfile': 'logs\\2019-03-18-18-08-53-mpf-RedTunic.log', 'pause': False, 'production': False, 'text_ui': True, 'loglevel': 10, 'consoleloglevel': 20, 'force_platform': None, 'syslog_address': None, 'mc_file_name': None}
2019-03-18 18:08:53,129 : INFO : Machine : MPF path: c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf
2019-03-18 18:08:53,129 : INFO : Machine : Machine path: C:\Users\Wilder\zelda
2019-03-18 18:08:53,129 : INFO : Machine : Platform: win32
2019-03-18 18:08:53,129 : INFO : Machine : Python executable location: c:\users\wilder\appdata\local\programs\python\python36\python.exe
2019-03-18 18:08:53,129 : INFO : Machine : Python version: 3.6.4 (64-bit)
2019-03-18 18:08:53,574 : INFO : Machine : Machine config file #1: config.yaml
2019-03-18 18:08:53,576 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\9bf880992bc5b85aa1bd2cc37b214e56.mpf_cache
2019-03-18 18:08:53,578 : DEBUG : asyncio : Using selector: SelectSelector
2019-03-18 18:08:53,579 : INFO : Machine : Initialise MPF.
2019-03-18 18:08:53,606 : DEBUG : FAST : Configuring FAST hardware.
2019-03-18 18:08:53,790 : INFO : FAST : Connecting to \\.\COM11 at 921600bps
2019-03-18 18:08:53,824 : INFO : FAST : Connected! Processor: NET, Board Type: FP-CPU-002-2, Firmware: 01.02
2019-03-18 18:08:53,824 : INFO : EventManager : Event: ======'machine_var_fast_net_firmware'====== Args={'value': '01.02', 'prev_value': None, 'change': True}
2019-03-18 18:08:53,824 : INFO : EventManager : Event: ======'machine_var_fast_net_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2019-03-18 18:08:53,824 : WARNING : FAST : Not resetting FAST NET because firmware is currently broken.
2019-03-18 18:08:53,888 : INFO : FAST : Connecting to \\.\COM12 at 921600bps
2019-03-18 18:08:53,936 : INFO : FAST : Connected! Processor: RGB, Board Type: FP-CPU-002-2, Firmware: 01.00
2019-03-18 18:08:53,937 : INFO : EventManager : Event: ======'machine_var_fast_rgb_firmware'====== Args={'value': '01.00', 'prev_value': None, 'change': True}
2019-03-18 18:08:53,937 : INFO : EventManager : Event: ======'machine_var_fast_rgb_model'====== Args={'value': 'FP-CPU-002-2', 'prev_value': None, 'change': True}
2019-03-18 18:08:53,937 : INFO : EventManager : Event: ======'machine_var_credits_string'====== Args={'value': 'FREE PLAY', 'prev_value': None, 'change': True}
2019-03-18 18:08:53,999 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\data/machine_vars.yaml
2019-03-18 18:08:54,001 : INFO : EventManager : Event: ======'machine_var_player1_score'====== Args={'value': 2, 'prev_value': None, 'change': True}
2019-03-18 18:08:54,001 : INFO : EventManager : Event: ======'machine_var_mpf_version'====== Args={'value': 'MPF v0.50.12', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,001 : INFO : EventManager : Event: ======'machine_var_mpf_extended_version'====== Args={'value': 'MPF v0.50.12, Config version:5, Show version: 5, BCP version:1.1', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,001 : INFO : EventManager : Event: ======'machine_var_python_version'====== Args={'value': '3.6.4', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,002 : INFO : EventManager : Event: ======'machine_var_platform'====== Args={'value': 'Windows-7-6.1.7601-SP1', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,002 : INFO : EventManager : Event: ======'machine_var_platform_system'====== Args={'value': 'Windows', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,002 : INFO : EventManager : Event: ======'machine_var_platform_release'====== Args={'value': '7', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,002 : INFO : EventManager : Event: ======'machine_var_platform_version'====== Args={'value': '6.1.7601', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,002 : INFO : EventManager : Event: ======'machine_var_platform_machine'====== Args={'value': 'AMD64', 'prev_value': None, 'change': True}
2019-03-18 18:08:54,002 : INFO : EventManager : Event: ======'init_phase_1'====== Args={}
2019-03-18 18:08:54,054 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\225554da95d8871f7382aa94a6d6b3eb.mpf_cache
2019-03-18 18:08:54,056 : DEBUG : ModeController : Loaded mode lost_woods_starter
2019-03-18 18:08:54,057 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\28e1731e516595966b3afcb08910cb63.mpf_cache
2019-03-18 18:08:54,059 : DEBUG : ModeController : Loaded mode heart_tracker
2019-03-18 18:08:54,059 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\072dbb4c30ca18f53440337aa5cd71bd.mpf_cache
2019-03-18 18:08:54,061 : DEBUG : ModeController : Loaded mode light_world
2019-03-18 18:08:54,062 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\dd39e2de39eb354a1ec99be2f3167765.mpf_cache
2019-03-18 18:08:54,064 : DEBUG : ModeController : Loaded mode heartmeter
2019-03-18 18:08:54,064 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\86e0d165ea95889c95b56598c3ebe263.mpf_cache
2019-03-18 18:08:54,067 : DEBUG : ModeController : Loaded mode music_player
2019-03-18 18:08:54,068 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\aecceeb5afbfdc7257044e8417ebcf98.mpf_cache
2019-03-18 18:08:54,074 : DEBUG : ModeController : Loaded mode attract
2019-03-18 18:08:54,074 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\6fd2b5e36bbbd9c12f9ee3aa32f1ceca.mpf_cache
2019-03-18 18:08:54,078 : DEBUG : ModeController : Loaded mode ball_hold_tracker
2019-03-18 18:08:54,078 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\c35ad2e595070cac0b6f696d6f7f6af6.mpf_cache
2019-03-18 18:08:54,081 : DEBUG : ModeController : Loaded mode ball_save
2019-03-18 18:08:54,081 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\fc30620fce8f82c18fed051cdde27ef9.mpf_cache
2019-03-18 18:08:54,084 : DEBUG : ModeController : Loaded mode 5waycombotracker
2019-03-18 18:08:54,084 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\05768db999abb9e30be16a432a0581c8.mpf_cache
2019-03-18 18:08:54,088 : DEBUG : ModeController : Loaded mode extraball
2019-03-18 18:08:54,088 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\b0ee6a771658ae3fb7e70c6d5ff59887.mpf_cache
2019-03-18 18:08:54,098 : DEBUG : ModeController : Loaded mode game
2019-03-18 18:08:54,098 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\4afc9f23ca6340163a8bb51d7deea78f.mpf_cache
2019-03-18 18:08:54,102 : DEBUG : ModeController : Loaded mode base
2019-03-18 18:08:54,102 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\e44f2e5e1ef00181aa9bc27df94a0c3d.mpf_cache
2019-03-18 18:08:54,105 : DEBUG : ModeController : Loaded mode escape_starter
2019-03-18 18:08:54,105 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\916c9e2d9177837f45e9cdbb7c1d0b93.mpf_cache
2019-03-18 18:08:54,108 : DEBUG : ModeController : Loaded mode unusual_show_tracker
2019-03-18 18:08:54,108 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\fd6423453738b6cee138971698f65579.mpf_cache
2019-03-18 18:08:54,111 : DEBUG : ModeController : Loaded mode mode_tracker
2019-03-18 18:08:54,111 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\58664c35241624b07b13025a5964d2a4.mpf_cache
2019-03-18 18:08:54,115 : DEBUG : ModeController : Loaded mode video_player
2019-03-18 18:08:54,115 : INFO : ConfigProcessor : Loading config from cache: C:\Users\Wilder\AppData\Local\Temp\1f8ff34a8859f8c62d77912f33e29b9c.mpf_cache
2019-03-18 18:08:54,118 : DEBUG : ModeController : Loaded mode 2ballqualifier
2019-03-18 18:08:54,118 : WARNING : ConfigProcessor : Config file in cache changed: C:\Users\Wilder\zelda\modes\highscore\config\highscore.yaml
2019-03-18 18:08:54,118 : INFO : ConfigProcessor : Loading config from file C:\Users\Wilder\zelda\modes\highscore\config\highscore.yaml.
2019-03-18 18:08:54,119 : DEBUG : YAML File Interface : Loading file: C:\Users\Wilder\zelda\modes\highscore\config\highscore.yaml
2019-03-18 18:08:54,153 : ERROR : asyncio : Exception in callback EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...column 1\'',)>) at c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py:96
handle: <Handle EventManager._async_handler_done(<QueuedEvent>)(<Task finishe...column 1\'',)>) at c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py:96>
Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 194, in load
    config = self.process(f)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 220, in process
    return yaml.load(data_string, Loader=MpfLoader)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\main.py", line 75, in load
    return loader.get_single_data()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\constructor.py", line 59, in get_single_data
    return self.construct_document(node)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\constructor.py", line 68, in construct_document
    for dummy in generator:
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\ruamel\yaml\constructor.py", line 481, in construct_yaml_map
    value = self.construct_mapping(node)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 135, in construct_mapping
    format(key, key_node.start_mark))
KeyError: 'Key "slide_player" was defined multiple times in config   in "C:\\Users\\Wilder\\zelda\\modes\\highscore\\config\\highscore.yaml", line 42, column 1'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\asyncio\events.py", line 145, in _run
    self._callback(*self._args)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\events.py", line 99, in _async_handler_done
    future.result()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 113, in load_modes
    self.machine.modes[mode] = self._load_mode(mode)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 262, in _load_mode
    config = self._load_mode_config(mode_string)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\mode_controller.py", line 180, in _load_mode_config
    store_to_cache=self.machine.options['create_config_cache'])
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 111, in load_config_files_with_cache
    ignore_unknown_sections)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\config_processor.py", line 131, in _load_config_file_and_return_loaded_files
    config = FileManager.load(filename, expected_version_str, True)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\file_manager.py", line 154, in load
    return interface.load(file, verify_version, halt_on_error)
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\file_interfaces\yaml_interface.py", line 208, in load
    raise ValueError(msg)
ValueError: Error found in file C:\Users\Wilder\zelda\modes\highscore\config\highscore.yaml: 'Key "slide_player" was defined multiple times in config   in "C:\\Users\\Wilder\\zelda\\modes\\highscore\\config\\highscore.yaml", line 42, column 1'
2019-03-18 18:08:54,399 : ERROR : root : Exception is not set.
Traceback (most recent call last):
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\commands\game.py", line 208, in __init__
    MachineController(mpf_path, machine_path, vars(self.args)).run()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 661, in run
    self.initialise_mpf()
  File "c:\users\wilder\appdata\local\programs\python\python36\lib\site-packages\mpf\core\machine.py", line 653, in initialise_mpf
    if init.exception():
asyncio.base_futures.InvalidStateError: Exception is not set.
